### ISSUE TITLE
[COLOR]: Foresee "Feedback" blue button MUST have sufficient contrast ratio

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#6340]
**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

The "Feedback" blue button shown throughout VA.gov **must** have sufficient color contrast. Currently, it does not pass the color contrast check in an axe scan:

```diff
! Element has insufficient color contrast of 2.84 (foreground color: #ffffff, 
! background color: #00a6d2, font size: 10.5pt (14px), font weight: normal). 
! Expected contrast ratio of 4.5:1
```

### Why it matters

Some people with low vision experience low contrast, meaning that there aren't very many bright or dark areas. Everything tends to appear about the same brightness, which makes it hard to distinguish outlines, borders, edges, and details. Text that is too close in luminance (brightness) to the background can be hard to read.

There are nearly three times more individuals with low vision than those with total blindness. One in twelve people cannot see the average full spectrum of colors - about 8% of men and 0.4% of women in the US. A person with low vision or color blindness is unable to distinguish text against a background without sufficient contrast.

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

**Success Criterion:** Ensure color contrast of at least 4.5:1 for small text or 3:1 for large text, even if text is part of an image. Large text has been defined in the requirements as 18pt (24 CSS pixels) or 14pt bold (19 CSS pixels). 

## Environment

* Server destination: production

## Steps to Recreate

1. Enter `https://www.va.gov/disability/` in browser
2. Run axe-coconut scan in the browser
3. Verify the color contrast issue shown in the Screenshots below

## WCAG or Vendor Guidance (optional)

* [Text elements must have sufficient color contrast against the background](https://dequeuniversity.com/rules/axe/3.4/color-contrast?application=AxeChrome)
* [WCAG 1.4.3](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html)

## Screenshots 

![ACS-Feedback-btn-contrast-issue](https://user-images.githubusercontent.com/57469/75595929-4b988800-5a5c-11ea-96a5-2b548b798325.png)

![ACS-Feedback-btn-contrast-issue-1](https://user-images.githubusercontent.com/57469/75595946-64a13900-5a5c-11ea-9807-642b1b53a558.png)

**Comment by alexpappasoddball** [comment]
@brandonrapp Adding to your backlog.  Let me know if you have questions

**Comment by timwright12** [comment]
Do we still need this now that Foresee is being removed?

**Comment by meganhkelley** [comment]
@jwoodman5 FYI

**Comment by jwoodman5** [comment]
@timwright12 We do not need this anymore. We should no longer have any ForeSee feedback buttons (surveys) on any VA.gov pages.
